src/test/java/com/ksm/wstbackoffice/WstBackofficeApplicationTests.java

```diff
@@ -1,13 +1,12 @@
 package com.ksm.wstbackoffice;
 
+import com.ksm.wstbackoffice.controller.BaseControllerIT;
 import org.junit.jupiter.api.Test;
-import org.springframework.boot.test.context.SpringBootTest;
 
-@SpringBootTest
-class WstBackofficeApplicationTests {
+class WstBackofficeApplicationTests extends BaseControllerIT {
```

Comment:
Please find out why we need it and give me the reason we need this test class. Otherwise, remove it.

Reply:
This class can be usefull for test some controllers or services method, but we did not use it because I removed